src/components/LineGrid/LineGrid.tsx

```diff
@@ -195,6 +196,9 @@ const LineGrid = (props: Props) => {
                 {renderLineKindDropdown()}
                 {renderSortDropdown()}
             </div>
+            <div className={styles.TopLine}>
```

Comment:
Nitpicking here but it's most typical to keep these lowercase. A better name for this might be `toplineWrapper` since it wraps that component, rather than being a part of it.